_data/simple-icons.json

```
@@ -964,6 +964,11 @@
             "hex": "21375A",
             "source": "https://appsignal.com/"
         },
+        {
+            "title": "Appsmith",
+            "hex": "FF6D2D",
+            "source": "https://github.com/appsmithorg/appsmith/blob/f7e74e17a2e205df77451572d7791eabd29340ec/app/client/src/assets/svg/appsmith_logo_primary.svg"
```

```suggestion
            "source": "https://www.appsmith.com/news"
```

They have press kit link in this page. We should use the icon style for Simple Icons. But there is no vector version of their brand in the press kit. We could ask them for help to upload a vector version to their Google Drive.